solr/core/src/test/org/apache/solr/security/JWTAuthPluginIntegrationTest.java

```diff
@@ -188,11 +181,13 @@ public void infoRequestValidateXSolrAuthHeadersBlockUnknownFalse() throws Except
         "  \"redirect_uris\":[],\n" +
         "  \"authorizationEndpoint\":\"http://acmepaymentscorp/oauth/auz/authorize\",\n" +
         "  \"client_id\":\"solr-cluster\"}", authData);
+    myCluster.shutdown();
   }
 
   @Test
   public void testMetrics() throws Exception {
-    configureClusterStaticKeys("jwt_plugin_jwk_security.json");
+    // Here we use the "global" class-level cluster variable, but not for the other tests
```

Comment:
I combined these two tests into one, using the global "cluster" var, since they both rely on validating metrics (which is bound to the global cluster)